bitcoin/script/miniscript.h

```diff
@@ -1232,6 +1232,7 @@ inline NodeRef<typename Ctx::Key> FromScript(const CScript& script, const Ctx& c
     using namespace internal;
     std::vector<std::pair<opcodetype, std::vector<unsigned char>>> decomposed;
     if (!DecomposeScript(script, decomposed)) return {};
+    if (decomposed.size() > 201) return {};
```

Comment:
I don't think that's exactly right, as the limit is on 201 non-push opcodes, but `decomposed` also includes the pushes.